src/figuresClasses.ts

```diff
@@ -1,11 +1,103 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'rectangle' | 'circle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  a: number;
+
+  b: number;
+
+  c: number;
+
+  constructor(
+    a: number,
+    b: number,
+    c: number,
+    public shape: 'triangle',
+    public color: 'red' | 'green' | 'blue',
+  ) {
+    this.a = a;
+    this.b = b;
+    this.c = c;
+
+    if (this.a <= 0 || this.b <= 0 || this.c <= 0) {
+      throw new Error('Length is not allowed to be lower than zero');
+    }
+
+    if (
+      this.a + this.b <= this.c ||
+      this.a + this.c <= this.b ||
+      this.b + this.c <= this.a
+    ) {
+      throw new Error('Measurements do not match');
```

Comment:
[CODE STYLE] - The error message 'Measurements do not match' is not very descriptive. It would be better to explain that the sum of the lengths of any two sides must be greater than the length of the third side.